SA/design_concept/content/poc/desktop_app.typ

```diff
@@ -0,0 +1,11 @@
+= Desktop Application
+A desktop application can be installed on the user's device with one installer.
+Both the frontend and backend are executed on the user's device.
+
+Pros:
+- A desktop-app doesn't require infrastructure to host it.
+- A desktop-app allows access to the local file system, allowing VisualFP to manage project files on the disk and use other development tools, such as Cabal #footnote("https://www.haskell.org/cabal/").
+
+Cons:
+- Every user has to install the application themselves.
+- The application has to support different operating systems.
```

Comment:
```suggestion
- The application must be build per operating system.
```

That the application supports different operating systems is part of the requirement